src/algebra/algebra/basic.lean

```diff
@@ -1252,26 +1252,6 @@ end algebra
 
 end ring
 
-section nat
-
-variables (R : Type*) [semiring R]
-
-section span_nat
-open submodule
-
-lemma span_nat_eq_add_group_closure (s : set R) :
-  (span ℕ s).to_add_submonoid = add_submonoid.closure s :=
-eq.symm $ add_submonoid.closure_eq_of_le subset_span $ λ x hx, span_induction hx
-  (λ x hx, add_submonoid.subset_closure hx) (add_submonoid.zero_mem _)
-  (λ _ _, add_submonoid.add_mem _) (λ _ _ _, add_submonoid.nsmul_mem _ ‹_› _)
-
-@[simp] lemma span_nat_eq (s : add_submonoid R) : (span ℕ (s : set R)).to_add_submonoid = s :=
-by rw [span_nat_eq_add_group_closure, s.closure_eq]
-
-end span_nat
-
-end nat
-
 section int
 
```

Comment:
It's probably worth moving `span_int_eq_add_group_closure` and `span_int_eq` below to the other file too, just to avoid confusion when people can't find them, leading to them being added again.

No need to golf their proofs in this PR to match the nat ones unless you want to.

Note you'll need  `{M : Type*} [add_comm_group M]` if you put it just below the `nat` ones, since otherwise you'll get only the `add_comm_monoid` instance.